webapp/src/core.ts

```diff
@@ -347,8 +347,14 @@ export function promptAsync(options: PromptOptions): Promise<string> {
     options.onLoaded = () => {
         let dialogInput = document.getElementById('promptDialogInput') as HTMLInputElement;
         if (dialogInput) {
-            dialogInput.focus();
```

Comment:
Reason this works is because setSelectionRange does the same thing. This might be legacy. Let's remove for now, but keep an eye out for any regressions in selection.